viam-cartographer/src/carto_facade/util.cc

```diff
@@ -83,7 +95,7 @@ int read_pcd(std::string pcd, pcl::PCLPointCloud2 &blob) {
             } break;
             // binary compressed
             case 2:
-
+                VLOG(1) << "parsing as binary compressed";
```

Comment:
Maybe we shouldn't log this given the log line that's directly below?

Reply:
done